samples/sampler/stories/default/ImageItem.js

```diff
@@ -0,0 +1,27 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {text, select} from '@enact/storybook-utils/addons/knobs';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import ImageItem, {ImageItemBase} from '@enact/agate/ImageItem';
+
+ImageItem.displayName = 'ImageItem';
+const Config = mergeComponentMetadata('ImageItem', ImageItem, ImageItemBase);
+
+storiesOf('Agate', module)
+	.add(
+		'ImageItem',
+		() => (
+			<div style={{width: '400px', height: '300px'}}>
```

Comment:
We prefer to use `scaleToRem`. Then the div element will display based on the resolution properly.

```
import ri from '@enact/ui/resolution';
...{width: ri.scaleToRem(400), ...}...
```